### PR TITLE
add badges to crates.io and docs.rs to README.md

### PR DESCRIPTION
**Pull request by wolthom**

### CODE REVIEW

**Comment by algesten** [comment]
Thanks for this!

The problem is that the readme is generated from the lib.rs - so any changes must be done there and then run `cargo readme`

**Comment by wolthom** [comment]
@algesten I tried to update the PR. If I understood `cargo-readme` correctly, I added the section to the `lib.rs` docstring and regenerated the `README.md` using `cargo readme > README.md`. Is that correct?

**Comment by algesten** [comment]
That's exactly right. Thanks!

I'll take a look.

File: src/lib.rs [hunk 1]
```diff
@@ -9,6 +9,25 @@
...
+//!  </a>
+//!  <!-- Docs -->
+//!  <a href="https://docs.rs/ureq">
+//!    <img src="https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square"
+//!      alt="docs.rs docs" />
+//!  </a>
+//!  <!-- Downloads -->
+//!  <a href="https://crates.io/crates/ureq">
+//!    <img src="https://img.shields.io/crates/d/ureq.svg?style=flat-square"
+//!      alt="Download" />
```
Comment (algesten) [inline src/lib.rs:27]: Should this alt-text say "Crates.io downloads" or something like that?

Comment (wolthom) [reply src/lib.rs]: Good suggestion, thank you! I updated the alt text and also slightly changed the badge positions to show up at the top. In my opinion this looks a bit cleaner.

**Comment by algesten** [comment]
Thanks

**Pull request merged**